tests/resources/system_tests/test_forever/launch_forever.py

```diff
@@ -0,0 +1,5 @@
+import time
+
+for i in range(10000):
```

Comment:
I feel we should make this loop forever. I don't want test to rely on this being 100 seconds long for cleanup.